community/contributing/docs/github-changes.md

```diff
@@ -10,7 +10,10 @@ sidebar:
 # Open a pull request (PR) on GitHub
 
 1. Click the **Suggest an Edit** link on the documentation page you want to update. This takes you to the page's source file in GitHub.
-1. Click the pencil icon to edit the file.
+1. Click the **Edit this file in the fork of your project** pencil icon to edit the file.
```

Comment:
Hmm, for me it just says "Edit this file"

![edit_this_file](https://user-images.githubusercontent.com/29312809/80819669-6c3ba600-8ba3-11ea-88ef-b76e76fbda01.png)

Reply:
Either way, I think it's fine just to say "**Edit this file** icon"

Reply:
That's bizarre. I'm on a Mac, and the hover help is the same regardless of browser.  I don't see the computer monitor icon in your screenshot either.
![edit-the-file](https://user-images.githubusercontent.com/26799583/80821482-6db69f80-8b9e-11ea-83d0-cfd9ca7d01f7.png)

Reply:
Changed to "Edit this file"

Reply:
@aimeeu do you have merge rights in the repo? It's possible that I'm seeing something different because I do. Thus you're seeing what the user will see, and I'm not. "Edit this file..." with the elipsis, seems like the safe choice.

Reply:
Nope - no merge rights. I'm just a lowly contributor. LOL.